static/src/js/components/GranuleResults/__tests__/GranuleResultsItem.test.js

```diff
@@ -335,316 +363,417 @@ function setup(type, overrideProps) {
+        })
 
-        removeButton.simulate('click')
         expect(props.onExcludeGranule.mock.calls.length).toBe(1)
         expect(props.onExcludeGranule.mock.calls[0]).toEqual([{
           collectionId: 'collectionId',
           granuleId: '170417722'
         }])
-
-        expect(removeButton.props().title).toEqual('Filter granule')
       })
     })
   })
 
   describe('when no granules are in the project', () => {
     test('does not have an emphisized or deepmphisized class', () => {
-      const { enzymeWrapper } = setup('cmr')
-      expect(enzymeWrapper.props().className).not.toContain('granule-results-item--emphisized')
-      expect(enzymeWrapper.props().className).not.toContain('granule-results-item--deemphisized')
+      setup('cmr')
+
+      const granuleResultsItem = screen.getByTestId('granule-results-item')
+      expect(granuleResultsItem.className).not.toContain('granule-results-item--emphisized')
+      expect(granuleResultsItem.className).not.toContain('granule-results-item--deemphisized')
     })
 
     test('displays the add button', () => {
-      const { enzymeWrapper } = setup('cmr')
-      const addButton = enzymeWrapper.find(Button)
-
-      expect(addButton.props().title).toContain('Add granule')
+      setup('cmr')
+      expect(screen.getByLabelText('Add granule')).toBeInTheDocument()
     })
   })
 
   describe('when displaying granules in the project', () => {
     describe('when displaying granule in the project', () => {
       test('displays the remove button', () => {
-        const { enzymeWrapper } = setup('override', {
+        setup('override', {
           isGranuleInProject: jest.fn(() => true),
           isCollectionInProject: true
         })
-        const addButton = enzymeWrapper.find(Button)
 
-        expect(addButton.props().title).toContain('Remove granule')
-        expect(enzymeWrapper.props().className).toContain('granule-results-item--emphisized')
+        expect(screen.getByLabelText('Remove granule')).toBeInTheDocument()
+
+        const granuleResultsItem = screen.getByTestId('granule-results-item')
+        expect(granuleResultsItem.className).toContain('granule-results-item--emphisized')
       })
     })
 
     describe('when displaying granule not in the project', () => {
       test('displays the add button', () => {
-        const { enzymeWrapper } = setup('override', {
+        setup('override', {
           isGranuleInProject: jest.fn(() => false),
           isCollectionInProject: true
         })
-        const addButton = enzymeWrapper.find(Button)
 
-        expect(addButton.props().title).toContain('Add granule')
-        expect(enzymeWrapper.props().className).toContain('granule-results-item--deemphisized')
+        expect(screen.getByLabelText('Add granule')).toBeInTheDocument()
+
+        const granuleResultsItem = screen.getByTestId('granule-results-item')
```

Comment:
Avoid using test ids, need to find a way to find this element in a way that better simulates how a user would use the app